iothub/device/src/Authentication/ClientAuthenticationWithRegistrySymmetricKey.cs

```diff
@@ -138,5 +152,10 @@ private void SetDeviceId(string deviceId)
 
             _deviceId = deviceId;
         }
+
+        private void SetModuleId(string moduleId)
+        {
+            _moduleId = moduleId;
```

Comment:
not checking null here for module Id as it's optional while passing to the ctor

Reply:
Probably should not be whitespace, though, right?

Reply:
Yes, good catch